string[] object_names
---
bool result